packages/components/storybook/webpack.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function( { config } ) {
+	config.module.rules.push( {
+		test: /stories\/index.jsx?$/,
```

Comment:
We need to match `/\/.+\.mdx$/` and `/\/stories\/.+\.js$/`.

Reply:
I improved the match, so its not just index file.

Testing out using the source-loader on `introduction.story.mdx` and it gave some weird results, I'm not sure if a different loader is needed for that format.